grasa_event_locator/migrations/0001_initial.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 2.2.6 on 2019-10-18 02:37
```

Comment:
Just curious. Do we want to commit migrations to the repo just yet? Or is it better for everyone on the team to manually do a `makemigrations` and `migrate`? My concern is shipping a migration file for a fresh install if it isn't needed. What do you think?